### PR TITLE
Create troubleshooting article for lock contention in .NET Framework applications (publish with 9.7)

### PR DESCRIPTION
**Pull request by JcolemanNR**
Creates a new article of interest for .NET Agent customers experiencing high lock contention in instrumented applications.

### CODE REVIEW

File: src/content/docs/apm/agents/net-agent/troubleshooting/lock-contention-appdomain-getdata.mdx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+In version 9.3 of the .NET agent, a new environment variable was introduced that disables the use of AppDomain storage by the .NET agent:
+```
+NEW_RELIC_DISABLE_APPDOMAIN_CACHING=true
+```
+
+**Warning:** While this environment variable does eliminate lock contention from agent calls to AppDomain.GetData(), the total overhead of the .NET agent is increased when this environment variable is enabled. In our testing this resulted in less locking, but lower maximum application throughput with the .NET agent attached to a test application. 
+
+We are extremely interested in any feedback from customers experiencing this issue who try this new configuration option. 
+
+(TODO: How should customers contact us?)
```
Comment (JcolemanNR) [inline src/content/docs/apm/agents/net-agent/troubleshooting/lock-contention-appdomain-getdata.mdx:30]: @elucus: What should I put in here for this?

File: src/content/docs/apm/agents/net-agent/troubleshooting/lock-contention-appdomain-getdata.mdx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+translate:
+  - jp
+metaDescription: Instructions and information about reducing AppDomain.GetData() lock contention for impacted .NET Framework applications 
+redirects:
+  - /docs/agents/net-agent/troubleshooting/lock-contention-appdomain-getdata
+---
+
+## Problem
+
+When the .NET agent is attached to high throughput **.NET Framework** applications, some customers have reported that thread lock contention can increase dramatically to a point that the application becomes non-responsive. Upon deeper analysis with process dumps, many threads can be blocked in calls to AppDomain.GetData() that are introduced by the .NET Agent.
```
Comment (angelatan2) [inline src/content/docs/apm/agents/net-agent/troubleshooting/lock-contention-appdomain-getdata.mdx:17]: Please reword:
some customers have reported that thread lock contention can increase dramatically to a point that the application becomes non-responsive.
**to**
a few customers reported observing high thread contention.  Upon  reviewing the process dumps and stack traces, many threads may be blocked by calls to AppDomain.GetData( ).

**Review by angelatan2 — CHANGES_REQUESTED** [review]

**Review by angelatan2 — DISMISSED** [review]

**Comment by barbnewrelic** [comment]
@JcolemanNR  Looks great! I made some very minor styleguide edits in your PR, also added it to the taxonomy so it will be visible when published. But I did have a question: This refers to v9.7 but I am not seeing a release note for 9.7. Is this right, or do we need to hold on publishing this until v9.7 is released? Thanks for confirming!

**Comment by JcolemanNR** [comment]
Hello @barbnewrelic, thanks for making the modifications! 

We had originally planned to release version 9.7 today, but have moved that Monday 4/4/2022. It would be great if we could time the release of this article with that.

**Comment by barbnewrelic** [comment]
@JcolemanNR Thanks for confirming! I had heard about that delay but just wanted to double check in case this piece might need to go out ahead of time. I will keep your PR with other related PRs for .NET 9.7 release so everything goes together. Thanks again for creating a helpful troubleshooting doc!

**Review by barbnewrelic — APPROVED** [review]
Will publish with .NET agent release

**Review by barbnewrelic — APPROVED** [review]
.NET release is ready, I'll include this with those other docs!

**Pull request merged**